.github/workflows/build-and-test-msi.yaml

```diff
@@ -80,15 +80,16 @@ jobs:
           role-session-name: windows-msi
           aws-region: ${{ secrets.WINDOWS_REGION }}
       - name: Remove Finch VM
-        timeout-minutes: 2
+        timeout-minutes: 5
```

Comment:
does timeout minute increase help in the sense we have evidence the process is timing out or it will just delay the finding.

Reply:
In cases where the step completed successfully, I see it running for at most 15seconds. When it fails its timing out after whatever timeout we configure delaying the finding. 

Will leave it at 2mins as increasing doesn't help in any way.